app/src/main/java/org/fossasia/openevent/app/core/faq/list/FaqListPresenter.java

```diff
@@ -75,4 +81,43 @@ private void listenChanges() {
     public List<Faq> getFaqs() {
         return faqs;
     }
+
+    public void deleteFaqWithId(long id, boolean forceReload) {
```

Comment:
Is there any need of forceReload parameter here

Reply:
Ya, it's not necessary. I'll remove it.